snapshot/doc/user-guide.md

```diff
@@ -0,0 +1,112 @@
+Volume Snapshots in Kubernetes
+=========================================
+
+This document describes current state of Volume Snapshot support in Kubernetes provided by external controller and provisioner. Familiarity with [Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/), [Persistent Volume Claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) and [Dynamic Provisioning](http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html) is recommended.
+
+# Introduction
+
+Many storage systems provide the ability to create "snapshots" of a persistent volume to protect against data loss. The external snapshot controller and provisioner provide means to use the feature in Kubernetes cluster and handle the volume snapshots through Kubernetes API.
+
+# Features
+
+* Create snapshot of a `PersistentVolume` bound to a `PersistentVolumeClaim`
+* List existing `VolumeSnapshots`
+* Delete existing `VolumeSnapshot`
+* Create a new `PersistentVolume` from an existing `VolumeSnapshot`
+* Supported `PersistentVolume` [types](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes):
+    * Amazon EBS
+    * GCE PD
+    * HostPath
```

Comment:
OpenStack Cinder

Reply:
Is OpenStack Cinder already supported for snapshots?

Reply:
yes, kudos @xing-yang and @j-griffith

Reply:
This is my fault. I will add Cinder.